### ISSUE TITLE
Exception "Why no lastCargoEntry?" at 2024-01-05 6:04:05 PM -08:00

### ISSUE THREAD

**Issue by DawnTreader** [njthomson/SrvSurvey#31]
### What happened?

loading after the main menu

### Version

1.0.0.0+e7a842ffaa2afeffa4d3be794dff444b015ead69

### Exception message

Why no lastCargoEntry?

### Exception stack

   at SrvSurvey.game.Game.initCargoData() in D:\a\SrvSurvey\SrvSurvey\SrvSurvey\game\Game.cs:line 660
   at SrvSurvey.game.Game.initializeFromJournal(LoadGame loadEntry) in D:\a\SrvSurvey\SrvSurvey\SrvSurvey\game\Game.cs:line 486
   at System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid2[T0,T1](CallSite site, T0 arg0, T1 arg1)
   at SrvSurvey.game.Game.Journals_onJournalEntry(JournalEntry entry, Int32 index) in D:\a\SrvSurvey\SrvSurvey\SrvSurvey\game\Game.cs:line 709

### Contact Details

dawntreader.ed@outlook.com

### Relevant log output

```shell
18:03:48: Game.initSystemData: rewind to last FSDJump or CarrierJump
18:03:48: Reading: Journal.2024-01-04T210150.01.log
18:03:48: Game.initSystemData: found last FSDJump, to 'Brairee FB-U d4-490' (16846732383659)
18:03:48: Loading SystemData for: 'Brairee FB-U d4-490' (16846732383659)
18:03:48: Nothing found for: 'Brairee FB-U d4-490' (16846732383659)
18:03:48: walkDeep: count: 2
18:03:48: Game.initSystemData: Processing 0 journal items forwards...
18:03:48: Game.initSystemData: complete 'Brairee FB-U d4-490' (16846732383659), bodyCount: 0
18:03:48: Game.initCargoData: rewind to last Cargo
18:03:48: Exception processing event 'LoadGame':
System.Exception: Why no lastCargoEntry?
   at SrvSurvey.game.Game.initCargoData() in D:\a\SrvSurvey\SrvSurvey\SrvSurvey\game\Game.cs:line 660
   at SrvSurvey.game.Game.initializeFromJournal(LoadGame loadEntry) in D:\a\SrvSurvey\SrvSurvey\SrvSurvey\game\Game.cs:line 486
   at System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid2[T0,T1](CallSite site, T0 arg0, T1 arg1)
   at SrvSurvey.game.Game.Journals_onJournalEntry(JournalEntry entry, Int32 index) in D:\a\SrvSurvey\SrvSurvey\SrvSurvey\game\Game.cs:line 709
```

**Comment by njthomson** [comment]
@DawnTreader thank you for reporting this - a fix is available, please try build `1.1.0.2` - https://github.com/njthomson/SrvSurvey/releases/tag/1.1.0.2

**Comment by njthomson** [comment]
Fix confirmed with build: https://github.com/njthomson/SrvSurvey/releases/tag/1.1.0.6
